lib/insights/api/common/rbac/access.rb

```diff
@@ -25,6 +25,23 @@ def accessible?
             @acl.any?
           end
 
+          def is_accessible?(app_name, resource, verb)
```

Comment:
For following methods, may we set default value `ENV["APP_NAME"]` for `app_name`?  They might be called multiple times in each app.  Add a check on `ENV["APP_NAME"].nil?` ?

Reply:
@hsong-rh I thought about that but it runs into problems about where the argument is

is_accessible?(resource, verb, app_name = ENV['APP_NAME'])

but most of the permissions look like

catalog:portfolios:read

Where app_name comes first.

In the case of catalog we have to check the permission for catalog and approval so it has to be passed in all the time.